user-interface/src/data-verification/ConsolidationCasesTable.tsx

```diff
@@ -45,7 +47,11 @@ function _ConsolidationCaseTable(
   }));
 
   return (
-    <table className="usa-table usa-table--borderless" id={id} data-testid={id}>
+    <table
+      className="usa-table usa-table--borderless consolidation-cases-table"
+      id={id}
+      data-testid={id}
+    >
```

Comment:
We don't need to do it here, but I think soon we should consider making a Table component to isolate the USWDS classes.

Reply:
💯